Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,42 @@
+.action{
+    color: #FFFFFF;
+    background-color: #9F001E;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1;
+    font-size: 2.3rem;
+    font-weight: 200;
+    width: 58px;
+    border: 0;
```

Comment:
You can give the buttons width and height through padding instead of having to hard code a width and make them a flex container